#!/usr/bin/env python2
from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "asr_ism_visualizations"


gen = ParameterGenerator()

gen.add("markerLifetime", double_t, 0, "Time Markers are shown (0=permanent).", 0, 0, 120)
gen.add("baseFrame", str_t, 0, "Base Frame of scene for poseprediction visualization.", "/map")
gen.add("lineScale", double_t, 0, "Width of lines for poseprediction visualization.", 0.002, 0, 0.01)
gen.add("axisScale", double_t, 0, "Lenght of coodinateaxis for poseprediction visualization.", 0.03, 0, 0.1)
gen.add("maxDistanceInOverlay", double_t, 0, "Maximum distance [m] in overlay for poseprediction visualization.", 0.1, 0, 0.5)
gen.add("maxAngleInOverlay", double_t, 0, "Maximum angle [DEG] in overlay for poseprediction visualization.", 15, 0, 30)

exit(gen.generate(PACKAGE, "asr_ism_visualizations", "ism_pose_prediction_visualizer"))
